php/docker/templates/nginx-conf.d/90-metrics.conf

```diff
@@ -60,32 +111,114 @@ server {
 
     location {{ default "/metrics/nginx" .Env.STACK_METRICS_NGINX_PATH }} {
         content_by_lua_block {
-            metric_connections:set(
+            metrics_nginx["http_connections"]:set(
                 ngx.var.connections_reading, {
                     ngx.var.server_name,
                     {{ $siteName }},
                     ngx.var.host,
                     "reading"
                 }
             )
-            metric_connections:set(
+            metrics_nginx["http_connections"]:set(
                 ngx.var.connections_waiting, {
                     ngx.var.server_name,
                     {{ $siteName }},
                     ngx.var.host,
                     "waiting"
                 }
             )
-            metric_connections:set(
+            metrics_nginx["http_connections"]:set(
                 ngx.var.connections_writing, {
                     ngx.var.server_name,
                     {{ $siteName }},
                     ngx.var.host,
                     "writing"
                 }
             )
-            prometheus:collect()
+            prometheus_nginx:collect()
+        }
+    }
+
+    location {{ default "/metrics/php-fpm" .Env.STACK_METRICS_PHP_PATH }} {
+    {{- if ne "off" ( default "/-/php-status" .Env.PHP_STATUS_PATH ) }}
+        content_by_lua_block {
+            local httpc = http.new()
+
+            local stats_url = "http://127.0.0.1:"
+              .. {{ default "8080" .Env.PORT | quote }}
+              .. {{ default "/-/php-status" .Env.PHP_STATUS_PATH | quote }}
+              .. "?json"
+
+            local res, err = httpc:request_uri(stats_url, { method = "GET" })
+
+            if not res then
+                ngx.status = ngx.HTTP_INTERNAL_SERVER_ERROR
+                ngx.say("Error: Failed to read FPM status at " .. stats_url)
+                return
+            end
+
+            local fpm_stats = cjson.decode(res.body)
```

Comment:
Can this fail if you receive a 200 response with empty body? How will it fail in that case?

Reply:
I've updated to handle parsing failure.